src/unistd.rs

```diff
@@ -441,6 +441,49 @@ pub fn mkdir<P: ?Sized + NixPath>(path: &P, mode: Mode) -> Result<()> {
     Errno::result(res).map(drop)
 }
 
+/// Creates new fifo special file (named pipe) with path `path` and access rights `mode`.
+///
+/// # Errors
+///
+/// There are several situations where mkfifo might fail:
+///
+/// - current user has insufficient rights in the parent directory
+/// - the path already exists
+/// - the path name is too long (longer than `PATH_MAX`, usually 4096 on linux, 1024 on OS X)
+///
+/// For a full list consult
+/// [man mkfifo(3)](http://man7.org/linux/man-pages/man3/mkfifo.3.html)
```

Comment:
Odd that you apparently develop on OSX but reference Linux man pages.  However, nix is cross-platform, so the best man pages to use are opengroup's.  Could you please change the link to this?
http://pubs.opengroup.org/onlinepubs/9699919799/utilities/mkfifo.html

Reply:
Well not that odd considering all other man page links in that file are pointing to http://man7.org/linux.

Reply:
Yeah, we've been slowly moving them as we go, but we haven't mass-converted.